### PR TITLE
Add documentation about config for java heap in API Mediation Layer services

### PR DESCRIPTION
**Pull request by pablocarle**
Describe your pull request here:

This PR adds an article about advanced configuration for the API Mediation Layer Java-based services.

List the file(s) included in this PR:

After creating the PR, follow the instructions in the comments.

### CODE REVIEW

File: docs/user-guide/api-mediation/configuration-customizing-java-heap-sizes.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Customizing Java Heap sizes
+
+:::info Role: system programmer
+::: 
+
+As a systems programmer, you can customize the Java heap memory size available for the API Mediation Layer components.
```
Comment (janan07) [inline docs/user-guide/api-mediation/configuration-customizing-java-heap-sizes.md:6]: Perhaps it would be useful to describe what "heap memory size" is...

**Review by janan07 — APPROVED** [review]
Reviewed and edited by doc squad

File: docs/user-guide/api-mediation/configuration-customizing-java-heap-sizes.md (hunk 1 above)
Comment (pablocarle) [reply docs/user-guide/api-mediation/configuration-customizing-java-heap-sizes.md:6]: Ok, I'll give it a try

**Review by balhar-jakub — APPROVED** [review]
It looks good, but should be against master branch as it applies to version 2.15 as well.

**Pull request merged**